src/party.cpp

```diff
@@ -419,7 +421,7 @@ void Party::shareExperience(uint64_t experience, Creature* source/* = nullptr*/)
 	leader->onGainSharedExperience(shareExperience, source);
 }
 
-bool Party::canUseSharedExperience(const Player* player) const
+bool Party::canUseSharedExperience(const Player* player)
```

Comment:
I remove ``const`` frm this member-function because I needed to modify the data member ``hasRequiredLevelToShare`` inside of it.